### ISSUE TITLE
Hide a NodePath from a Plugin

### ISSUE THREAD

**Issue by RodrigoMedeirosrRS** [tobspr/RenderPipeline#68]
Hello!

It's me, bothering you again, so now my problem is with Motion Blur:
![captura de tela de 2016-12-30 13-20-20](https://cloud.githubusercontent.com/assets/7956669/21572440/1eb77baa-ced9-11e6-8e93-b22583fba634.png)

So i wank know if is possible hide sone Nodepah from one plugin, in this case, the motion Blur. How you can see into image above, the are applying a string distortion into the main character because he is always in front of camera. So i want hide the character Nodepath only from motionblur plugin.

**Comment by tobspr** [comment]
Hi!

The actual problem is that the velocity of the character is computed wrong.
This is because panda does not provide previous frames transforms, which
the render pipeline would need to accurately reconstruct the last frames
position.

I have a Patch for panda that I want to merge soon, which covers this
issue.
Until then, I believe there is no other option than to disable motion blur,
until it is fixed.

Best regards,
Tobias

Am 30.12.2016 21:49 schrieb "Rodrigo Medeiros Lehnemann" <
notifications@github.com>:

Hello!

It's me, bothering you again, so now my problem is with Motion Blur:
[image: captura de tela de 2016-12-30 13-20-20]
<https://cloud.githubusercontent.com/assets/7956669/21572440/1eb77baa-ced9-11e6-8e93-b22583fba634.png>

So i wank know if is possible hide sone Nodepah from one plugin, in this
case, the motion Blur. How you can see into image above, the are applying a
string distortion into the main character because he is always in front of
camera. So i want hide the character Nodepath only from motionblur plugin.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/tobspr/RenderPipeline/issues/68>, or mute the thread

**Comment by RodrigoMedeirosrRS** [comment]
Right, i will wait for this patch, please, post here a notice when you release it.

... and have a happy new year :)

**Comment by tobspr** [comment]
I will notice you when I merged it :) 

Thanks, you too!

**Comment by tobspr** [comment]
Just letting you know, I'm currently a bit busy due to exams and other stuff, I will probably only be able to fix this in a month or two.

**Comment by RodrigoMedeirosrRS** [comment]
No problem, we will wait for the fix ;)
